[2022-03-22 00:00:02,594] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.enter_virtual_env 2022-03-21T18:00:00+00:00 [queued]>
[2022-03-22 00:00:02,600] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: matchup.enter_virtual_env 2022-03-21T18:00:00+00:00 [queued]>
[2022-03-22 00:00:02,600] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-22 00:00:02,600] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2022-03-22 00:00:02,600] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-22 00:00:02,607] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): enter_virtual_env> on 2022-03-21T18:00:00+00:00
[2022-03-22 00:00:02,610] {standard_task_runner.py:52} INFO - Started process 447103 to run task
[2022-03-22 00:00:02,617] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'matchup', 'enter_virtual_env', '2022-03-21T18:00:00+00:00', '--job-id', '420', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/matchup.py', '--cfg-path', '/tmp/tmp9hhu5y3h', '--error-file', '/tmp/tmppr9yij35']
[2022-03-22 00:00:02,618] {standard_task_runner.py:77} INFO - Job 420: Subtask enter_virtual_env
[2022-03-22 00:00:02,650] {logging_mixin.py:104} INFO - Running <TaskInstance: matchup.enter_virtual_env 2022-03-21T18:00:00+00:00 [running]> on host 531edfb863fc
[2022-03-22 00:00:02,679] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=r.chesne@gmail.com
AIRFLOW_CTX_DAG_OWNER=Robin
AIRFLOW_CTX_DAG_ID=matchup
AIRFLOW_CTX_TASK_ID=enter_virtual_env
AIRFLOW_CTX_EXECUTION_DATE=2022-03-21T18:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-21T18:00:00+00:00
[2022-03-22 00:00:02,680] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2022-03-22 00:00:02,681] {bash.py:158} INFO - Running command: . /app/env/bin/activate
[2022-03-22 00:00:02,691] {bash.py:169} INFO - Output:
[2022-03-22 00:00:02,698] {bash.py:177} INFO - Command exited with return code 0
[2022-03-22 00:00:02,718] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=matchup, task_id=enter_virtual_env, execution_date=20220321T180000, start_date=20220322T000002, end_date=20220322T000002
[2022-03-22 00:00:02,743] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-03-22 00:00:02,785] {local_task_job.py:146} INFO - Task exited with return code 0
